ui/ducks/send/send.js

```diff
@@ -1042,13 +1166,18 @@ const slice = createSlice({
         state.account.balance = action.payload.nativeBalance;
         state.asset.balance = action.payload.assetBalance;
         state.gas.gasLimit = action.payload.gasLimit;
-        state.gas.gasPrice = action.payload.gasPrice;
+        slice.caseReducers.updateGasFeeEstimates(state, {
+          payload: {
+            gasFeeEstimates: action.payload.gasFeeEstimates,
+            gasEstimateType: action.payload.gasEstimateType,
+          },
+        });
         state.gas.gasTotal = action.payload.gasTotal;
         state.gas.gasEstimatePollToken = action.payload.gasEstimatePollToken;
         if (action.payload.gasEstimatePollToken) {
           state.gas.isGasEstimateLoading = false;
         }
-        if (state.stage !== SEND_STAGES.UNINITIALIZED) {
+        if (state.stage !== SEND_STAGES.INACTIVE) {
```

Comment:
TODO: understand the conditions under which stage in `INACTIVE` vs not inactive

Reply:
- [ ] Understand this

Reply:
The hardcoded initial state has the stage set to `INACTIVE`. It gets changed from this when:
- the `initializeSendState` think is fulfilled
- when the recipient is updated (including when it is first set)
- when beginning to edit a transcaction

I don't think there is a signficant difference between the `UNITIALIZED` and `INACTIVE` names